Developers/Default.html

```diff
@@ -0,0 +1,91 @@
+<!doctype html>
+
+<html>
+
+<head >
+<link rel="stylesheet" type="text/css" href="Style_sheet.css">
+<title> Що треба? </title> 
+
+<meta charset="utf-8">
+<meta name="description" content="" />
+<meta name="keywords" content="m" />
+<meta http-equiv="author" content="" />
+<meta http-equiv="content-language" content="ukr-UA" />
+</head>
+
+
+<body style="font-family: Comic Sans, Cochin, Georgia, Times, 'Times New Roman', serif;">
+
+
+<header> 
+    <br>
+    <p style="font-size: 25px;"> Що треба? </p>
+    <a style="color: white;" href="file:///C:/Ucustud/ITshnik/HTML_page/Developers/About.html?">Про нас</a>
+
+</header>
+
+<aside> 
+    <br>
+    <form  action="" methods="">
+    
+        <label for="login"> Логін: </label>
+        <input type="text" size="10px" id="login" name=""></br>
+    
+        <label for="Password"> Пароль: </label>
+        <input type="text" size="10px" id="Password" name="">
+    
+    
+    </form >
+    <a style="color: blue;" href="file:///C:/Ucustud/ITshnik/HTML_page/Developers/Register.html?"> Зареєструватись </a>
+    <br>
+    <br>
+    
+    
+
+    <form action="" methods=""> 
+        Ваше місто: </br>
+        <select>
+            <option value="Львів"> Львів </option>
+            <option value="Київ"> Київ </option>
+            <option value="Харків"> Харків </option>
+        </select>
+    </form>
+
+    <br>
+    <br>
+    
+    <form action="" methods=""> 
+        Виберіть категорію:
+        <select>
+            <option value="Їжа"> Їжа </option>
+            <option value="Одяг"> Одяг </option>
+        </select>

```

Comment:
Maybe better to use radio-buttons here to make it possible to select several options.